src/convertResults.ts

```diff
@@ -1,21 +1,31 @@
-import { CodacyResult } from "codacy-seed"
+import { FileError, Issue, ToolResult } from "codacy-seed"
 import { CLIEngine, Linter } from "eslint"
-import { flatMap } from "lodash"
 
 import { blacklist } from "./blacklist"
 import { patternIdToCodacy } from "./model/patterns"
 
-export function convertResults(report: CLIEngine.LintReport): CodacyResult[] {
-  return flatMap(report.results, result => {
+export function convertResults(report: CLIEngine.LintReport): ToolResult[] {
+  const results: ToolResult[] = []
+  report.results.forEach((result) => {
     const filename = result.filePath
-    const pairs = result.messages
-      .filter(r => r.ruleId && !blacklist.includes(r.ruleId))
-      .map(m => [m.ruleId, m]) as [string, Linter.LintMessage][]
-    return pairs.map(([ruleId, m]) => {
-      const line = m.line
-      const message = m.message
-      const patternId = patternIdToCodacy(ruleId)
-      return new CodacyResult(filename, message, patternId, line)
-    })
+    const messages = result.messages
+    if (
+      messages.length === 1 &&
+      messages[0].ruleId == null &&
+      messages[0].message.startsWith("Parsing error:")
```

Comment:
Is every error message starting with this message?

Reply:
I don't know!
The problem is that some plugins are using `null` as `ruleId` so I have to add this constrain to match at least default ESLint parsing errors.